nucypher/characters/control/specifications/bob.py

```diff
@@ -45,15 +41,15 @@ class Retrieve(BaseSchema):
             type=click.STRING,
             required=False)
     )
-    message_kit = character_fields.UmbralMessageKit(
+    message_kit = character_fields.MessageKit(
         required=True,
         load_only=True,
         click=options.option_message_kit(required=False)
     )
```

Comment:
Are we going to keep the REST API to only one message kit for the character control, or a list of message kits? For `Bob.retrieve()`python API it takes a list of message kits now.

If desired we can do it in a separate PR if need be.

Reply:
Technically, it did take a list before as well, it's just that it was spliced. That is, it was `retrieve(kit1, kit2, ...)`, now it's `retrieve([kit1, kit2, ...], ...)`. For Porter it probably makes sense to support batch retrieve.

Reply:
Yep - but currently the Bob character control specification here for the `/retrieve` endpoint only accepts a single kit and not multiple - do we want it to only allow for 1 kit to be passed?

Porter uses a batch retrieve i.e. a list of retrieval kits can currently be provided - https://github.com/derekpierre/nucypher/blob/porter-retrieval/nucypher/utilities/porter/control/specifications/porter_schema.py#L135.

Reply:
Would it be better done as a part of #2768?

Reply:
Sure - I can add it in